samples/algorithms/integer-factorization/FlameGraphResourcesEstimator.cs

```diff
@@ -12,10 +12,10 @@
 
 namespace Microsoft.Quantum.Simulation.Simulators
 {
-    public class FlameGraphResourcesEstimator : ResourcesEstimator
+    public class FlameGraphResourcesEstimator : QCTraceSimulator
```

Comment:
While it makes sense that this resolves the compilation error, is this actually valid given what the sample is? Does `FlameGraphResourceEstimator` produce valid/usable results when run through the tracer? If not, maybe the whole sample should just be removed?

Reply:
In this particular instance no additional functionality of ResourcesEstimator is used. QCTraceSimulator works just as well. But see other notes.